department-of-veterans-affairs/abd-vro
# Delete unused docs directory

## What was the problem?

We use wiki for documentation and this directory is contributing to clutter. If there is anything we want to keep it should be migrated to the wiki.

## How does this fix it?[^1]

* Remove the docs directory

## How to test this PR
- No functional code change

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
